src/facade/dragonfly_connection.cc

```diff
@@ -751,11 +734,11 @@ void Connection::DispatchFiber(util::FiberSocketBase* peer) {
   cc_->conn_closing = true;
 
   // Clean up leftovers.
-  DispatchCleanup clean_op;
   while (!dispatch_q_.empty()) {
+    // we actually need to have the element so that the dtor would be called
```

Comment:
I am not sure what happens here. I would assume that `pop_front();` would be enough.

Reply:
I will check this to make sure that this is not required any more

Reply:
turn out even the loop is not required - "clear" is enough